src/plugins/plugin.legend.js

```diff
@@ -44,8 +45,13 @@ defaults._set('global', {
 			// lineDashOffset :
 			// lineJoin :
 			// lineWidth :
+			// pointStyle: symbol use on legend when usePointStyle is set
+			// legendSymbol : Symbol use on legend if not usePointStyle
+			// boxWidth : width of legend symbol if not userPointStyle
```

Comment:
`userPointStyle` should be `usePointStyle`. 
I would also reword these to something like:
```js
			// pointStyle: symbol used on legend when 'usePointStyle' is true
			// legendSymbol : symbol used on legend when usePointStyle is false
			// boxWidth : width of legend symbol when usePointStyle is false
```